### DISCUSSION TITLE
Updating the documentation

### DISCUSSION THREAD

**Discussion by altonius** [davidusb-geek/emhass#265] [Q&A] [answered]
Hiya David,
**Firstly a big thanks for your efforts in maintaining such an amazing project that delivers so much.**
I'm not a developer, but I'm happy to help out with updating and improving the projects documentation. 
Before I get started I've got a couple of quick questions:

1. Would you prefer a big PR with all the changes, or done on a page by page basis?
2. How attached are you to:
    - the current structure
    - the language used in the document (i.e. the documentation currently says "we do this", "we do that")
4. How would you prefer I ask questions about a feature I'm not using so that I can correctly update things?

Cheers

**Comment by davidusb-geek** [comment, marked as answer]
Hi and thanks!
For the questions:
1. Probably a first PR to see concretely what you mean and then a big PR if needed.
2. For the structure. Our main readme is way too long. It is confusing in part because of the difference installation methods. I'm not attached to the current structure and there is a lot of room for improvements. But at the same time we have come a long way to what it is today. But open and happy to discuss proposals for a new structure and new sections, or a rearrange to make it more readable yes. For the language part, yes definitely let's improve that. 
3. I guess that discussion can be directly in the PR
